addons/udes_stock/tests/common.py

```diff
@@ -0,0 +1,467 @@
+# -*- coding: utf-8 -*-
+from odoo.tests import common
+
+
+@common.at_install(False)
+@common.post_install(True)
+class BaseUDES(common.SavepointCase):
```

Comment:
If this is defining the base class for testing, this should be one of first commits.

Reply:
It was how commits are shown in github, since this is the second commit in reality.